### ISSUE TITLE
Enhancement: New voice clips for Tankman in Stress

### ISSUE THREAD

**Issue by sawyer400** [ninjamuffin99/Funkin#1419]
So like, i mean depending on health, Tankman says something different.
Like this, Over 30 health: "Heh, pretty good!"
Under 30 health: "Heh, you suck!"

**Comment by VMGuy23** [comment]
still pretty good because you not dead

**Comment by DeceptedParasite** [comment]
this would actually spice things up if this gets added, would make some people salty yeah, but it would be so funny

**Comment by Cheemsandfriends** [comment]
@DeudzXD Better health because who cares about accuracy?

**Comment by Steviegt6** [comment]
FNF's health system just serves as a way to determine if you're dead or not. There's no real "rhythmatic" value to it. Accuracy would make more sense if it was properly recorded and used in the base game.

**Comment by ninjamuffin99** [comment]
actually i change my mind, we will keep this in our heads

**Comment by polybiusproxy** [comment]
hehe

**Comment by Windows1Door2Daniel** [comment]
Would be better if he says 

> How did manage to survive rigth here?

**Comment by shibest** [comment]
Yes true, voice clips should change depending on accuracy, and perhaps other features can also be based on accuracy

**Comment by Windows1Door2Daniel** [comment]
I marked my own comment as spam

But it would be better use healt
